app/src/main/resources/swagger/api-docs.json

```diff
@@ -8473,6 +8716,65 @@
           }
         }
       },
+      "TaxonomyResource" : {
+        "title" : "TaxonomyResource",
+        "required" : [ "ecType", "ecTypeCode", "endDate", "legalReasonCollection", "macroAreaDescription", "macroAreaEcProgressive", "macroAreaName", "serviceType", "serviceTypeCode", "serviceTypeDescription", "specificBuiltInData", "startDate", "taxonomyVersion" ],
+        "type" : "object",
+        "properties" : {
+          "ecType" : {
+            "type" : "string",
+            "description" : "Entity creditor type"
+          },
+          "ecTypeCode" : {
+            "type" : "string",
+            "description" : "Entity creditor type code"
+          },
+          "endDate" : {
+            "type" : "string",
+            "description" : "End date of validity"
+          },
+          "legalReasonCollection" : {
+            "type" : "string",
+            "description" : "Legal reason for collection"
+          },
+          "macroAreaDescription" : {
+            "type" : "string",
+            "description" : "Macro area description"
+          },
+          "macroAreaEcProgressive" : {
+            "type" : "string",
+            "description" : "Macro Area Progressive"
+          },
+          "macroAreaName" : {
+            "type" : "string",
+            "description" : "Macro area name"
+          },
+          "serviceType" : {
+            "type" : "string",
+            "description" : "Service type"
+          },
+          "serviceTypeCode" : {
+            "type" : "string",
+            "description" : "Service type code"
+          },
+          "serviceTypeDescription" : {
+            "type" : "string",
+            "description" : "Service description"
+          },
+          "specificBuiltInData" : {
+            "type" : "string",
+            "description" : "Specific collection data"
+          },
+          "startDate" : {
+            "type" : "string",
```

Comment:
It's better if the date format is specified in the swagger

Reply:
can you explain the date format please?

Reply:
in the taxonomy get api the startDate type is string end we exopse the same type